### PR TITLE
Adding the early draft of Tables content.

### PR DESCRIPTION
**Pull request by Serdaro**

### CODE REVIEW

File: docs/modules/getting-started/pages/tables.adoc [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+and historical data in the historical mode.
+
+The current mode is always available. The historical mode requires xref:deploy-manage:historical-metrics.adoc[] to be enabled.
+
+The date/time input controls what data is displayed. The input is disabled if only current data is available.
+You can select any arbitrary point in the past to see the historical data from that period. You can drag
+the chart to adjust the time point.
+
+Tables, by default, show up to 1 minute of data.
+In the current mode it requests data in chunks every 5 seconds, and concatenates it.
```
Comment (promanenko) [inline docs/modules/getting-started/pages/tables.adoc:20]: not sure `concatenates` is correct here. We don't append data every 5 seconds, it's a sliding window

File: docs/modules/getting-started/pages/tables.adoc [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+In the historical mode it requests 1 minutes of data up to the selected time point once.
```
Comment (promanenko) [inline docs/modules/getting-started/pages/tables.adoc:21]: In historical mode it requests up to 1 day of data in a given range. User can choose a time range with from/to widgets

File: docs/modules/getting-started/pages/tables.adoc [hunk 3]
```diff
@@ -0,0 +1,40 @@
...
+When the historical mode is activated, the date/time input displays the selected time point. You can press the *RESET TIME* button
+to switch back to the current mode.
+
+image:ROOT:WidgetHistorical.png[alt=Historical mode, align="center"]
+
+== Presets
+
+Tables provide access to an extensive list of metrics
+and use a selection of filters to show only relevant data matching a certain condition on the table.
+A preset is a combination of metrics and a filter. You can add/edit new presets and remove the existing ones.
```
Comment (promanenko) [inline docs/modules/getting-started/pages/tables.adoc:34]: there are no filters in table preset, just a list of columns

**Review by promanenko — APPROVED** [review]
LGTM

**Review by jgardiner68 — COMMENTED** [review]
Improvements to table docs

**Pull request merged**